PR: https://github.com/odoo/odoo/pull/235040

From: ba149f981182fcb4877c722ab9a032200aa3d23a
From: pusu-odoo
Date: 2026-02-17 14:21:44

Breaking data model changes scores: del:3 + add:0, change matches:
-    instruction_type = fields.Selection([
-    instruction_pdf = fields.Binary('PDF')
-    instruction_google_slide = fields.Char('Google Slide', help="Paste the url of your Google Slide. Make sure the access to the document is public.")

Total Changes: 6

[IMP] maintenance: simplify instructions tab layout

* Currently the `Instructions` tab combines three different instruction types
(PDF, Google Slides, and plain text) in a single block. The layout doesn’t look
good and makes it hard for users to manage instructions.

These changes simplifies the `Instructions tab` by keeping only the text input
(removed `PDF` and `Google Slide` options). Users are now expected to upload any
related documents directly within the instructions content.

TaskId-5150425

Part-of: odoo/odoo#235040
Related: odoo/enterprise#99156
Related: odoo/upgrade#8819
Signed-off-by: Tiffany Chang (tic) <tic@odoo.com>

================================= pseudo patch: =================================

--- a/addons/maintenance/models/maintenance.py
+++ b/addons/maintenance/models/maintenance.py
@@ -240,12 +240,6 @@ class MaintenanceRequest(models.Model):
     maintenance_team_id = fields.Many2one('maintenance.team', string='Team', required=True, index=True, default=_get_default_team_id,
                                           compute='_compute_maintenance_team_id', store=True, readonly=False, check_company=True)
     duration = fields.Float(help="Duration in hours.", compute='_compute_duration', store=True)
-    instruction_type = fields.Selection([
-        ('pdf', 'PDF'), ('google_slide', 'Google Slide'), ('text', 'Text')],
-        string="Instruction", default="text"
-    )
-    instruction_pdf = fields.Binary('PDF')
-    instruction_google_slide = fields.Char('Google Slide', help="Paste the url of your Google Slide. Make sure the access to the document is public.")
     instruction_text = fields.Html('Text')
     recurring_maintenance = fields.Boolean(string="Recurrent", compute='_compute_recurring_maintenance', store=True, readonly=False)
     repeat_interval = fields.Integer(string='Repeat Every', default=1)
